pages/partials/newVariantInfo.ejs

```diff
@@ -0,0 +1,13 @@
+<p>This assessment is set up to allow you to try multiple variants of
+the same question. Each of these variants is equivalent to the
```

Comment:
I think it's best to make this per-question, because even on a homework some other questions might be `singleVariant`.
```suggestion
<p>This question allows you to try multiple variants.
Each of these variants is equivalent to the
```